File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //declare variable height
    int height;

    //Make sure number isn't < or > 1 & 8
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //creating a new line with integer line
    for (int line= 0; line < height; line++)

    {
        //creating space
        for (int space = height - line - 1; space > 0; space--)
        {
            printf(" ");
    }
        //creating the first hash pyramid
        for (int left_hash = 0; left_hash < line + 1; left_hash++)
        {
            printf("#");
        }
        //creating the gap between pyramids
        printf("  ");

        //creating second hash pyramid
        for (int right_hash = 0; right_hash < line + 1; right_hash++)
        {
            printf("#");
        }
        //end of the sequence
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# loop for prompting the user for a height between 1 and 8
while True:
    height = get_int("height: ")
    # if the height is correct break out
    if height > 0 and height < 9:
        break

# building the pyramid
for line in range(1, (height + 1), 1):
    print(" " * (height - line), end="")
    print("#" * line, end="")
    print(" " * 2, end="")
    print("#" * line, end="")
    print()

